src/index.js

```diff
@@ -26,21 +26,24 @@ async function bundleMDX(
     esbuildOptions = options => options,
     globals = {},
     cwd = path.join(process.cwd(), `__mdx_bundler_fake_dir__`),
+    grayMatterOptions = options => options
   } = {},
 ) {
+  /* c8 ignore start */
```

Comment:
There is no way to test this really so I'm excluding it from coverage, no need for an unhappy codecov bot.